templates/tfengine/components/cicd/env/main.tf

```diff
@@ -12,64 +12,79 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
```

Comment:
this file can be called cicd/triggers/main.tf

and your current triggers/main.tf can be renamed to trigger/main.tf

Reply:
Are you sure? The `triggers/main.tf` actually has more than 1 trigger.

Is just a semantic problem, not a big deal to leave it as you request, but I want to make you aware

Reply:
discussed offline, we can keep as is.